### DISCUSSION TITLE
MHO-C601 support

### DISCUSSION THREAD

**Discussion by binfill3** [pvvx/ATC_MiThermometer#428] [General]
Hi, I am getting ready to ditch my aqara temperature sensors, and this project is highly recommended on the home assistant forums. I see that Xiaomi Miaomiaoce (MHO-C401) is listed as a supported device, but on aliexpress, I am able to find only MHO-C601. I was curious about this because of the eink display. Is the updated C601 supported?

Also, while we are here, which variant would you folks recommend for overall stability? The aqara sensors like to drop off the network every odd month. Thanks to improvements to core ZHA, these events no longer take down my zigbee network, but I am looking for a hassle free replacement for these.

**Comment by pvvx** [comment]
MHO-C601 does not have a Bluetooth chip.

**Comment by binfill3** [comment]
Duh, that's going to be a problem. Thank you for saving me some hassle! I'll go for the tried and tested Mijia versions.
